libraries _PLUGIN145 ITHO FAN/Itho/IthoCC1101.cpp

```diff
@@ -344,25 +341,26 @@ void IthoCC1101::initReceive()
 void  IthoCC1101::initReceiveMessage2(IthoMessageType expectedMessageType)
 {
 	uint8_t marcState;
-	
+
 	writeCommand(CC1101_SIDLE);	//idle
-	
-	//set datarate	
-	writeRegister(CC1101_MDMCFG4 ,0x5A); // set kBaud
+
+	//set datarate
+	writeRegister(CC1101_MDMCFG4 ,0x9A); // set kBaud
 	writeRegister(CC1101_MDMCFG3 ,0x83); // set kBaud
 	writeRegister(CC1101_DEVIATN ,0x50);
-	
+
  	//set fifo mode with fixed packet length and sync bytes
 	writeRegister(CC1101_PKTLEN ,42);			//42 bytes message (sync at beginning of message is removed by CC1101)
 	receiveState = ExpectNormalCommand;
-	
+
 	//set fifo mode with fixed packet length and sync bytes
 	writeRegister(CC1101_PKTCTRL0 ,0x00);
-	writeRegister(CC1101_SYNC1 ,170);			//message2 byte6
+	writeRegister(CC1101_SYNC1 ,170);			//message2 byte6 -- if not working change to 170, use 172 for remote with 'niet-thuis' function
```

Comment:
Is this something that is likely to not work on some nodes?
If so, then either use a define at the beginning of the source file (so you don't have to search in the code) or make it a configuration setting.

Reply:
Correct, there are currently two known-good values depending on the type of remote. I can declare it at the beginning of the file. I also made a version of the plugin where it could be changed in the plug-in configuration and it was passed to this function as a variable. This would be even easier for users. However, I am not sure if this is the best way to do it. Any suggestions on how to best pass this from the plugin to the functions in the ibrary?